src/main/java/Terminator.java

```diff
@@ -0,0 +1,409 @@
+
+    /**
+     * Global Array List of all tasks created by the user.
+     */
+    public static ArrayList<Task> tasksList = new ArrayList<Task>();
+
+    /**
+     * Given user input, extract Task Name and UserInput.
+     * @param userinput The string given by the user.
+     * @param eventType The type of task to extract based on.
+     * @return String array containing userinput and eventType.
+     */
+    private static String[] extractNameDateTime(String userinput, String eventType){
+        String[] returnArray = new String[2];
+        if (Objects.equals(eventType, DEADLINE_TYPE)){
+            // Get indexes to substring
+            int startOfByIndex = userinput.indexOf(BY_KEYWORD);
+            int endOfByIndex = startOfByIndex + 4;
+            int endOfDeadlineStringIndex = userinput.indexOf(DEADLINE_KEYWORD) + 9;
+
+            // Get specific task_name and date_time
+            String task_name = userinput.substring(endOfDeadlineStringIndex, startOfByIndex).strip();
+            String date_time = userinput.substring(endOfByIndex).strip();
+
+            // Place values into array
+            returnArray[TASK_NAME_INDEX] = task_name;
+            returnArray[DATE_TIME_INDEX] = date_time;
+        } else if (Objects.equals(eventType, EVENT_TYPE)){
+            // Get indexes to substring
+            int startOfAtIndex = userinput.indexOf(AT_KEYWORD);
+            int endOfAtIndex = startOfAtIndex + 4;
+            int endOfEventStringIndex = userinput.indexOf(EVENT_KEYWORD) + 6;
+
+            // Get specific task_name and date_time
+            String task_name = userinput.substring(endOfEventStringIndex, startOfAtIndex).strip();
+            String date_time = userinput.substring(endOfAtIndex).strip();
+
+            // Place values into array
+            returnArray[TASK_NAME_INDEX] = task_name;
+            returnArray[DATE_TIME_INDEX] = date_time;
+        } else {
+            // Extract values for ToDo
+            int endOfToDoStringIndex = userinput.indexOf(TODO_KEYWORD) + 5;
+            String task_name = userinput.substring(endOfToDoStringIndex).strip();
+            returnArray[TASK_NAME_INDEX] = task_name;
+
+        }
+        return returnArray;
+    }
+
+    /**
+     * Prints response back to user of task that is modified.
+     * @param task_number The index of the task to be updated.
+     */
+    private static void printUpdateMessage(int task_number) {
+        Task current_task = tasksList.get(task_number);
+        System.out.println("Great! The following item has been marked as completed" + System.lineSeparator() +
+                current_task.toString());
+        System.out.println(formatWithHeading("Is there anything else you would like me to do?",
+                TERMINATOR_FORMATTING));
+    }
+
+    /**
+     * Update the completion status of the Task to true.
+     * @param task_number The index of the task to be updated.
+     */
+    private static void updateCompletion(int task_number) {
```

Comment:
Yes, I agree that a better method name can be used. Completion seems abit vague.